CMake/Dependencies/libkvsCommonLws-CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsCommonLws-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           8bd5adc78e8453e52a8707748c3469d8c9b74c29
+    GIT_TAG           add-new-log-level
```

Comment:
Need to change to a specific commit once this PR passes CI: https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/pull/379